src/stories/components/404/not-found-component-404.tsx

```diff
@@ -0,0 +1,308 @@
+  marginBottom: '64px',
+  marginLeft: '32px',
+  marginRight: '32px',
+  [lightTheme.breakpoints.between('table_834', 'desktop_1194')]: {
+    marginBottom: '160px',
+    width: '580px',
+    height: '340px',
+  },
+  [lightTheme.breakpoints.between('desktop_1194', 'desktop_1280')]: {
+    marginBottom: '37.26px',
+    width: '675px',
+    height: '397.74px',
+  },
+  [lightTheme.breakpoints.between('desktop_1280', 'desktop_1440')]: {
+    marginBottom: '37.26px',
+    width: '675px',
+    height: '397.74px',
+  },
+  [lightTheme.breakpoints.between('desktop_1440', 'desktop_1920')]: {
+    marginBottom: '37.26px',
+    width: '675px',
+    height: '397.74px',
+  },
+  [lightTheme.breakpoints.up('desktop_1920')]: {
+    marginBottom: '37.26px',
+    width: '675px',
+    height: '397.74px',
+  },
+});
+
+const ContainerText = styled.div({
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'center',
+  alignItems: 'center',
+  paddingRight: '32px',
+  paddingLeft: '32px',
+  maxWidth: '343px',
+  [lightTheme.breakpoints.between('table_834', 'desktop_1194')]: {
+    maxWidth: '100%',
+  },
+  [lightTheme.breakpoints.between('desktop_1194', 'desktop_1280')]: {
+    maxWidth: '100%',
+  },
+  [lightTheme.breakpoints.between('desktop_1280', 'desktop_1440')]: {
+    maxWidth: '100%',
+  },
+  [lightTheme.breakpoints.between('desktop_1440', 'desktop_1920')]: {
+    maxWidth: '100%',
+  },
+  [lightTheme.breakpoints.up('desktop_1920')]: {
+    maxWidth: '100%',
+  },
+});
+
+const TextUps = styled(Typography)<{ isLight?: boolean }>(({ isLight }) => ({
+  fontFamily: 'FT Base, sans-serif',
+  fontStyle: 'normal',
+  fontWeight: 700,
+  fontSize: '36px',
+  lineHeight: '43px',
+  letterSpacing: '0.4px',
+  color: isLight ? '#787A9B' : '#D2D4EF',
+  textAlign: 'center',
+  marginBottom: '24px',
+  [lightTheme.breakpoints.between('table_834', 'desktop_1194')]: {
+    fontWeight: 700,
+    fontSize: '48px',
+    lineHeight: '58px',
+    marginBottom: '32px',
+  },
+  [lightTheme.breakpoints.between('desktop_1194', 'desktop_1280')]: {
+    fontWeight: 700,
+    fontSize: '48px',
+    lineHeight: '58px',
+    marginBottom: '32px',
+  },
+  [lightTheme.breakpoints.between('desktop_1280', 'desktop_1440')]: {
+    fontWeight: 700,
+    fontSize: '48px',
+    lineHeight: '58px',
+    marginBottom: '32px',
+  },
+  [lightTheme.breakpoints.between('desktop_1440', 'desktop_1920')]: {
+    fontWeight: 700,
+    fontSize: '48px',
+    lineHeight: '58px',
+    marginBottom: '32px',
+  },
+  [lightTheme.breakpoints.up('desktop_1920')]: {
+    fontWeight: 700,
+    fontSize: '48px',
+    lineHeight: '58px',
+    marginBottom: '32px',
+  }
+}));
+
+const TextDescription = styled(Typography)<{ isLight?: boolean }>(({ isLight }) => ({
+  fontFamily: 'FT Base,san-serif',
+  fontStyle: 'normal',
+  fontWeight: 500,
+  fontSize: '20px',
+  lineHeight: '24px',
+  textAlign: 'center',
+  letterSpacing: '0.4px',
+  color: isLight ? '#ADAFD4' : '#ADAFD4',
```

Comment:
This `TextDesciption` has the same color for light and dark themes, is it ok? If it is maybe it is a good idea just to use `color: #ADAFD4`

Reply:
My mistake, already fixed

Reply:
fixed